test/index.ts

```diff
@@ -1,22 +1,69 @@
-import { describe, it } from 'mocha';
+import { describe, it, beforeEach } from 'mocha';
 import { run } from '../src/server';
-import { expect } from 'chai';
 import { setupEnv } from '../src/environment';
+import { use as chaiUse, expect } from 'chai';
+import chaiExclude from 'chai-exclude';
 import axios from 'axios';
+import { appDataSource } from '../src/data-source';
+import { User } from '../src/entity/user';
+import { compare } from 'bcrypt';
+
+let serverUrl: string;
+
+chaiUse(chaiExclude);
 
 before(async () => {
   setupEnv();
   await run();
+  serverUrl = `http://localhost:${process.env.PORT}/`;
 });
 
-describe('Queries', () => {
-  it('should respond to a hello query', async () => {
-    const res = await axios.post(`http://localhost:${process.env.PORT}/`, {
-      query: 'query Query { hello }',
-      variables: {},
-      operationName: 'Query',
+describe('Query', () => {
+  describe('hello', () => {
+    it("should respond with 'wassup?'", async () => {
+      const res = await axios.post(serverUrl, {
+        query: 'query Query { hello }',
+        variables: {},
+        operationName: 'Query',
+      });
+
+      expect(res.data).to.deep.equal({ data: { hello: 'wassup?' } });
     });
+  });
+});
+
+describe('Mutation', () => {
+  const userRepository = appDataSource.getRepository(User);
 
-    expect(res.data).to.deep.equal({ data: { hello: 'wassup?' } });
+  describe('createUser', () => {
+    beforeEach(() => {
+      userRepository.delete({});
+    });
```

Comment:
aqui precisa ser assíncrono, para garantir que apagou antes do primeiro it executar